server/routes/userRoutes.js

```diff
@@ -16,15 +16,15 @@ userRouter.route('/signup')
     checkUserInput, checkUserInvalidDetails, emailExist,
     usernameExist, signUp
   );
+userRouter.route('/')
+  .get(isLoggedIn, getUser);
 
 userRouter.route('/login')
   .post(login);
 
-
 userRouter.route('/editprofile')
   .put(
-    isLoggedIn, checkUserInvalidDetails, emailExist, verifyUserIdExist,
```

Comment:
You used a lot of middlewares which helps your code to be cleaner and more organized. It is a good practice. Keep it up.